experiments/aldp/train.py

```diff
@@ -15,7 +15,7 @@
 from fab.utils.numerical import effective_sample_size
 from fab.utils.replay_buffer import ReplayBuffer
 from fab.utils.prioritised_replay_buffer import PrioritisedReplayBuffer
-from fab.core import P_SQ_OVER_Q_TARGET_LOSSES
+from fab.core import ALPHA_DIV_TARGET_LOSSES
```

Comment:
I'm not super confident in editing the aldp file - could you please check I've done it correctly?